src/Compute/Compute.Test/Compute.Test.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Compute" Version="24.1.0" />
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="2.4.2" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.9.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="21.0.0-preview" />
```

Comment:
Please use generate SDK from the previous PR (https://github.com/Azure/azure-powershell/pull/8713, versioned `20.0.0-preview`). It's used in multiple PRs so re-using it would help to avoid merge conflicts

Reply:
will change